iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -86,6 +86,15 @@ <h3 data-i18n="inspector.filter">Filter</h3>
             <option value="summationOrContributor" data-i18n="inspector.summationOrContributor">Summation or Contributor</option>
           </select>
 
+          <div class="control-label" data-i18n="inspector.dimensions">
+              Dimension Type
+          </div>
+          <select id="search-filter-dimension-type">
```

Comment:
Any reason not to make this a multi-select so they can filter the union of explicit + typed dimensions?

Reply:
Sure, 🤷 so I don't see why not. More useful yet is probably something like:

- [ ] None
- [ ] Explicit
   - [ ] Dim1
   - [ ] Dim2
- [ ] Typed
    - [ ] Dim1
    - [ ] Dim2

Reply:
I am going to proceed as is on this and we can circle back on this in the future, I'll create an issue to follow up

Reply:
https://github.com/Workiva/ixbrl-viewer/issues/431